src/DurableTask.Core/TaskActivityDispatcher.cs

```diff
@@ -173,6 +173,9 @@ async Task OnProcessWorkItemAsync(TaskActivityWorkItem workItem)
                         eventToRespond = new TaskFailedEvent(-1, scheduledEvent.EventId, e.Message, details);
                         this.logHelper.TaskActivityFailure(orchestrationInstance, scheduledEvent.Name, (TaskFailedEvent)eventToRespond, e);
                         CorrelationTraceClient.Propagate(() => CorrelationTraceClient.TrackException(e));
+
+                        traceActivity?.SetTag("otel.status_code", "ERROR");
+                        traceActivity?.SetTag("otel.status_description", e.Message + "\n" + details);
```

Comment:
I suggest we have a more condensed error description:
```suggestion
                        traceActivity?.SetTag("otel.status_description", $"{e.GetType().FullName}: {e.Message}");
```

In the above example, this would result in "__System.InvalidOperationException: This is an invalid operation__", which I think is enough information for distributed tracing. The full exception information (call-stack, etc.) should be in the existing logs anyways. I don't want to use `details` because that's an internal, serialized representation of the exception that we should avoid logging.

Reply:
That makes sense. I tried `e.GetType().FullName` and it resolves to `DurableTask.Core.Exceptions.TaskFailureException`. I then tried `e.InnerException.GetType().FullName` and it resolved to `System.InvalidOperationException`. I'll replace the existing `otel.status_description` value with `$"{e.InnerException.GetType().FullName}: {e.Message}`. 

I'll also make the same change in the orchestration exception code section to remove `details`.

<img width="959" alt="image" src="https://user-images.githubusercontent.com/15795068/158876633-9bdb71e3-da53-4a43-8b36-ca52c6a57aa9.png">

<img width="958" alt="image" src="https://user-images.githubusercontent.com/15795068/158876703-e7d02d23-cff4-48fb-972a-01365cfaa14e.png">